robottelo.properties.sample

```diff
@@ -388,6 +388,16 @@ port_range=9091, 14999
 # default value is private
 # managed_ip=Private
 
+#[gce]
+# Google Provider as Compute Resource
+# Project name in Google provider
+#project_id=sat6qe
```

Comment:
any specific reason to keep `sat6qe` value for `project_id` ?

Reply:
No specific reason, its a project created in GCE and its the name of a project there and not being created from Satellite.

Reply:
@jyejare could you please remove that from here as that is more looking a harcode way for same.

Reply:
Its not mandatory to keep everything private, this value serves as a default value to project_id param.